cmd/server.go

```diff
@@ -53,7 +51,7 @@ var serverCmd = &cobra.Command{
 			log.Fatalf("Cannot init db: %+v", err)
 		}
 
-		auth.InitGithubAuth(startup.Github)
+		auth.GitHubOAuth(startup.Github)
```

Comment:
You can delete this line since `GitHubOAuth()` has no side-effects and the result is unused.